README.md

```diff
@@ -11,3 +12,21 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+
+To prove this, we will use the change of base formula for logarithms:
+
+$\log_2(n) = \frac{\log_5(n)}{\log_5(2)}$
+
+Now, substituting this into the inequality, we get:
+
+$c_1 \cdot \frac{\log_5(n)}{\log_5(2)} \leq \log_5(n) \leq c_2 \cdot \frac{\log_5(n)}{\log_5(2)}$
```

Comment:
Not sure what you're doing here, but it looks like you're assuming specific $f(n)$ and $T(n)$, which you can't do for a general proof.